### ISSUE TITLE
"Bio Balance Detector" repo is not indexed (yet)

### ISSUE THREAD

**Issue by andrasfuchs** [KicadSearch/leopart#1]
[My repo](https://github.com/andrasfuchs/BioBalanceDetector) is not indexed yet, could you let me know what I need to do to get indexed?

**Comment by LasseMoench** [comment]
@andrasfuchs The GitHub crawling is not very good right now. The way it currently works is that it searches for repositories that have 'kicad' or 'pcb' in the readme file. If you add those words to your readme somewhere, you should get indexed the next time GitHub is crawled. I am also thinking of adding a way to manually submit repositories.

**Comment by andrasfuchs** [comment]
@LasseMoench Thanks for the quick reply! It has both words in the readme, so I guess I just have to wait a little longer ;)
I think the manual submit would be a nice addition.

**Comment by LasseMoench** [comment]
Hi @andrasfuchs, I had a look again and your repo is already indexed :) You can find it if you search for https://leopart.org/?q=ATTINY861-20SU since it is the only project including that version of the attiny861 (it seems)

**Comment by andrasfuchs** [comment]
@LasseMoench Thank you for checking it out again!

The license isn't right in the search result though: here is the MIT license I use 
https://github.com/andrasfuchs/BioBalanceDetector/blob/master/LICENSE
The result shows jQuery's MIT.

Also, I use AD7616 for example in one of my prototypes, but [it's not listen by leopart](https://leopart.org/?q=AD7616).

Here is the file that contains the component:
https://github.com/andrasfuchs/BioBalanceDetector/blob/a6b7948eddbebfaa39940ce1dec812c009e12678/Hardware/Schematics/BBDProto06/Venus-256/CellSlave.sch#L108

**Comment by LasseMoench** [comment]
Yes, you are right, we are currently crawling the repo for a license.md file, but GitHub actually has a fixed license url for each repo, which we should use. I will fix that. Just to be sure: You mean you're using GPL, not MIT, right?
I'll also look into why not all of your parts are indexed. Our parser for KiCad files is a little fragile, and if anything goes wrong during parsing we just ignore the part. Thanks for providing feedback that will help me improve the parser!

**Comment by andrasfuchs** [comment]
Yes, sorry, I meant GPL.
Thanks for your work, I think your search engine is a needed, excellent project!

**Comment by andrasfuchs** [comment]
@LasseMoench 
Could you check the problem with the search of "AD7616" again, please?
As I pointed out earlier, I use AD7616 in one of the projects in my repo, but your search engine doesn't return it as a hit.
I think it's still a bug :)

**Comment by LasseMoench** [comment]
Hi @andrasfuchs 
it is indeed a problem with the parser for kicad files. Your repo is found but the parser crashes for some reason. Unfortunately, this semester at uni is a lot more work than expected due to the current situation (studying from home apparently requires more mandatory exercises :) ), which is why I have not been able to work on this project in a long time. I hope I will be able to look into the issue this weekend.

**Comment by andrasfuchs** [comment]
Thanks @LasseMoench that would be great!
